gramps/gen/errors.py

```diff
@@ -138,6 +138,18 @@ def __str__(self):
         return self.value
 
 
+class NoneObjectError(HandleError):
```

Comment:
Would `AccessDeniedError` be more descriptive?

Reply:
Yes, much better. Thanks!